### DISCUSSION TITLE
Higher CPU/GPU usage on the main menu than in-game

### DISCUSSION THREAD

**Discussion by victorleblais** [voxeltycoon/issues#575] [General]
**Describe the bug**
I noticed that the game uses way more CPU and GPU resources when I'm idling on the main menu compared to when I'm actually in-game on a relatively big save.

**In-Game:**
![image](https://user-images.githubusercontent.com/4365075/61732359-bcb4e600-ad7d-11e9-972a-cbc3afc46eaf.png)
CPU usage: 25%
GPU usage: 10%

**Main menu**
![image](https://user-images.githubusercontent.com/4365075/61732334-b0308d80-ad7d-11e9-8df2-a9e334a4216b.png)
CPU usage: 50%
GPU usage: 90%

CPU:
![image](https://user-images.githubusercontent.com/4365075/61732229-6e074c00-ad7d-11e9-8045-f471d9d0af20.png)

GPU:
![image](https://user-images.githubusercontent.com/4365075/61732315-a149db00-ad7d-11e9-8b9d-aecec931ec0a.png)

**Steps to reproduce**
Just launching the game is enough

**Build version**
0.74 2019-07-19 ITCH PRE-ALPHA

**Platform**
Windows 10 Pro N 64-bit
Intel i7-8700k, 6 cores, 12 threads at 4.4Ghz
32GB of RAM, DDR4, 3466Mhz
GTX 1080Ti

**Reproducibility**
100% of the time.
Restarting the game changes nothing.
Entering and exiting any save changes nothing, same for asset editor.

**Comment by drewtato** [comment]
Is the framerate uncapped? Because it could be running the menu at 100FPS and the game at like 30.

**Comment by Toys0125** [comment]
I just tested on my pc with v-sync on my pc would use more resources then if just capped and when unlimited then I have the different usage like it shows you have.
Using bandicam framerate counter I would sit at 450 fps in the main menu then around 120 fps in-game with a lower amount of usage.

**Comment by Toys0125** [comment]
it's also funny that with v-sync on my GPU usage is around 60% when off but capped it's at 20%

**Comment by Toys0125** [comment]
I have:
Windows 10 Home Build 18362
Intel I7-5820k at 4.3 Ghz
32GB of Ram, DDR4, 3200Mhz
GTX 1080 Driver Version	26.21.14.3160

**Comment by Toys0125** [comment]
I also notice your uses more memory than mine as well after a quit of a game.
https://i.imgur.com/mlRyQ5c.png
